tests/Query/Filter/TagsField.test.ts

```diff
@@ -631,8 +631,9 @@ describe('Sort by tags', () => {
 describe('grouping by tag', () => {
     it('supports grouping methods correctly', () => {
         const field = new TagsField();
-        const fieldGrouper = field.createGrouper();
         expect(field.supportsGrouping()).toEqual(true);
+
+        const fieldGrouper = field.createGrouper();
```

Comment:
This one is curiously inconsistent with the others....

Reply:
oops =) fixed! =)